packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMainInput.tsx

```diff
@@ -68,17 +89,81 @@ function MaxButton(props: React.ButtonHTMLAttributes<HTMLButtonElement>) {
   )
 }
 
-function TransferPanelInputField(
-  props: React.InputHTMLAttributes<HTMLInputElement>
-) {
-  const { value = '', ...rest } = props
+function TokenBalance({
```

Comment:
please rename this because we have the old `TokenBalance` component that is still in use on the destination network box

https://github.com/OffchainLabs/arbitrum-token-bridge/blob/master/packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain/TokenBalance.tsx